deploy/helm/scf/assets/operations/instance_groups/api.yaml

```diff
@@ -125,7 +82,7 @@
   value: ((deployment-name))-doppler
 - type: replace
   path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/loggregator?/internal_url
-  value: http://((deployment-name))-log-api:8081
+  value: https://((deployment-name))-log-api:8081
```

Comment:
This is not an HTTPS server: https://github.com/cloudfoundry/capi-release/blob/1.83.0/jobs/cloud_controller_ng/spec#L1097.

Reply:
The server is HTTP/2.